code/api/src/modules/user/resolvers.js

```diff
@@ -77,3 +77,47 @@ export async function remove(parentValue, { id }) {
 export async function getGenders() {
   return Object.values(params.user.gender)
 }
+
+// Update user
+export async function update(parentValue, { id, style }) {
```

Comment:
This giant function takes the passed in `style: "Classic, Classic, Vintage, Vintage, Vintage, Hip Hop"` argument, changes it into a hash with a count of how many times it's in the `style` string, evaluates the hash, and eventually resolves with a `"Vintage but Classic with a touch of Hip Hop"` string.

Reply:
This is a very readable function, and requires very little explanation. Nice job! There are small refactoring opportunities that could make the function shorter, if you wish to do so, but it's also a non-issue. Glad to go over the options.